tests/src/test/java/tachyon/shell/command/ChgrpCommandTest.java

```diff
@@ -41,4 +41,24 @@ public void chgrpTest() throws IOException, TachyonException {
     group = mFileSystem.getStatus(new TachyonURI("/testFile")).getGroupName();
     Assert.assertEquals("group2", group);
   }
+
+  @Test
+  public void chgrpRecursiveTest() throws IOException, TachyonException {
+    clearLoginUser();
+    FileSystemTestUtils.createByteFile(mFileSystem, "/testDir/testFile", WriteType.MUST_CACHE, 10);
+    // "chgrp -R group1 /testDir" should apply to both dir and child file
+    mFsShell.run("chgrp", "-R", "group1", "/testDir");
+    String group = mFileSystem.getStatus(new TachyonURI("/testDir")).getGroupName();
+    Assert.assertEquals("group1", group);
+    group = mFileSystem.getStatus(new TachyonURI("/testDir/testFile")).getGroupName();
+    Assert.assertEquals("group1", group);
+    // chgrp to another group.
+    mFsShell.run("chgrp", "-R", "group2", "/testDir");
+    group = mFileSystem.getStatus(new TachyonURI("/testDir/testFile")).getGroupName();
+    Assert.assertEquals("group2", group);
+    // Also test "chgrp group3 /testDir -R"
```

Comment:
Make it clearer what's actually being tested here, e.g. 'Test that the "-R" option still works when it comes after the arguments'